src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -37,7 +37,9 @@ object ReadyValidIO {
 
     /** Indicates if IO is both ready and valid
      */
-    def fire(): Bool = target.ready && target.valid
+    def fire: Bool = target.ready && target.valid
```

Comment:
I think there is something in the docs that use `fire`. Which makes me wonder if we need to update the mdoc as part of this PR

Reply:
I checked and there are no warnings in the mdoc coming from this PR

Reply:
cool